py3-cairo.yaml

```diff
@@ -114,23 +127,6 @@ subpackages:
             python: python3.13
             import: ${{vars.import}}
 
-  - name: py3-cairo-dev
-    description: py3-cairo dev
-    dependencies:
-      runtime:
-        - cairo-dev
-        - py3-cairo
-    pipeline:
-      - uses: split/dev
-    test:
-      pipeline:
-        - uses: test/pkgconf
-        - uses: py/one-python
-          with:
-            content: |
-              include="$(python3 -c 'import ${{vars.import}}; print(${{vars.import}}.get_include())')"
-              test -f "$include/py3cairo.h"
```

Comment:
The reason for this package was to keep the `/usr/include/pycairo/py3cairo.h` file around. I'm not sure if anything uses it, but it could be a regression. The test doesn't seem appropriate though.

Reply:
then we should just straight out test for that path. because cairo.get_include() returns `/usr/lib/python3.13/site-packages/cairo/include`

```
$ apk add py3.13-cairo
...
(26/27) Installing cairo (1.18.2-r1)
(27/27) Installing py3.13-cairo (1.27.0-r2)
$  python3.13 -c 'import cairo; print(cairo.get_include())'
/usr/lib/python3.13/site-packages/cairo/include
```

Reply:
i think it makes most sense to just drop this 'py3-cairo-dev' package. if something needs it then we can add it. nothing in the archive right now seems to.